src/test/java/core/basesyntax/OperationStrategyTest.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax;
+
+import static org.junit.Assert.assertSame;
+
+import core.basesyntax.model.Operation;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.OperationStrategyImpl;
+import core.basesyntax.strategy.operations.BalanceHandler;
+import core.basesyntax.strategy.operations.OperationHandler;
+import core.basesyntax.strategy.operations.PurchaseHandler;
+import core.basesyntax.strategy.operations.ReturnHandler;
+import core.basesyntax.strategy.operations.SupplyHandler;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Before;
+import org.junit.Test;
+
+public class OperationStrategyTest {
+    private String operationLine;
+    private OperationStrategy strategy;
+
+    @Before
+    public void setUp() {
+        Map<Operation, OperationHandler> operationHandlerMap = new HashMap<>();
+        operationHandlerMap.put(Operation.BALANCE, new BalanceHandler());
+        operationHandlerMap.put(Operation.PURCHASE, new PurchaseHandler());
+        operationHandlerMap.put(Operation.RETURN, new ReturnHandler());
+        operationHandlerMap.put(Operation.SUPPLY, new SupplyHandler());
```

Comment:
I think it's better to move it to BeforeAll(BeforeClass) method, because we have the same strategy map for all tests, so we don't need to create it before each test